index.js

```diff
@@ -23,21 +26,63 @@ Object.keys(ansiStyles).forEach(function (key) {
 
 	styles[key] = {
 		get: function () {
-			return build.call(this, this._styles ? this._styles.concat(key) : [key]);
+			var codes = ansiStyles[key];
+			return build.call(this, this._styles ? this._styles.concat(codes) : [codes], key);
+		}
+	};
+});
+
+ansiStyles.color.closeRe = new RegExp(escapeStringRegexp(ansiStyles.color.close), 'g');
```

Comment:
Also, [it's not applied globally](https://github.com/chalk/ansi-styles/blob/master/index.js#L119-L122). We fixed that a while back :)